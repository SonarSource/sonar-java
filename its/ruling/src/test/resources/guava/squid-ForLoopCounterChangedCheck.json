{
'com.google.guava:guava:src/com/google/common/base/Ascii.java':[
442,
491,
],
'com.google.guava:guava:src/com/google/common/base/CharMatcher.java':[
1279,
],
'com.google.guava:guava:src/com/google/common/base/Utf8.java':[
94,
],
'com.google.guava:guava:src/com/google/common/math/IntMath.java':[
216,
],
'com.google.guava:guava:src/com/google/common/math/LongMath.java':[
260,
],
}
