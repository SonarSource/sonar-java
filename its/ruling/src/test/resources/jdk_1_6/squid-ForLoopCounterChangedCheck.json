{
'jdk_1_6:java/awt/Container.java':[
1355,
1373,
],
'jdk_1_6:java/awt/datatransfer/SystemFlavorMap.java':[
447,
449,
454,
],
'jdk_1_6:java/awt/font/TextJustifier.java':[
87,
],
'jdk_1_6:java/awt/font/TextMeasurer.java':[
349,
393,
],
'jdk_1_6:java/io/ObjectOutputStream.java':[
1962,
1972,
2121,
2204,
],
'jdk_1_6:java/io/RandomAccessFile.java':[
1075,
1076,
],
'jdk_1_6:java/lang/AbstractStringBuilder.java':[
1393,
],
'jdk_1_6:java/lang/Character.java':[
2631,
2633,
2671,
2673,
],
'jdk_1_6:java/lang/ConditionalSpecialCasing.java':[
196,
198,
],
'jdk_1_6:java/lang/String.java':[
1769,
],
'jdk_1_6:java/math/BigInteger.java':[
1283,
1284,
1456,
1470,
2671,
2671,
2672,
2674,
2675,
3139,
3139,
3140,
3142,
3143,
],
'jdk_1_6:java/math/MutableBigInteger.java':[
236,
238,
543,
559,
],
'jdk_1_6:java/net/URI.java':[
1695,
1700,
1703,
1706,
2314,
2680,
2757,
2766,
2766,
2767,
],
'jdk_1_6:java/net/URLEncoder.java':[
213,
244,
248,
],
'jdk_1_6:java/text/Bidi.java':[
236,
236,
],
'jdk_1_6:java/text/ChoiceFormat.java':[
177,
],
'jdk_1_6:java/text/DecimalFormat.java':[
1994,
1996,
2001,
2042,
2046,
2052,
2385,
2403,
2411,
2424,
2440,
2533,
2538,
2550,
2555,
],
'jdk_1_6:java/text/DigitList.java':[
294,
],
'jdk_1_6:java/text/MessageFormat.java':[
429,
],
'jdk_1_6:java/text/RBTableBuilder.java':[
515,
559,
],
'jdk_1_6:java/text/SimpleDateFormat.java':[
690,
758,
761,
881,
883,
884,
894,
1243,
1245,
1246,
1261,
],
'jdk_1_6:java/util/Arrays.java':[
1168,
1170,
1294,
1296,
],
'jdk_1_6:java/util/Calendar.java':[
1289,
],
'jdk_1_6:java/util/Collections.java':[
767,
],
'jdk_1_6:java/util/Hashtable.java':[
365,
],
'jdk_1_6:java/util/Random.java':[
167,
],
'jdk_1_6:java/util/concurrent/LinkedBlockingDeque.java':[
931,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
846,
847,
862,
863,
],
'jdk_1_6:java/util/regex/Pattern.java':[
1163,
1174,
1179,
1309,
2894,
2896,
],
}
