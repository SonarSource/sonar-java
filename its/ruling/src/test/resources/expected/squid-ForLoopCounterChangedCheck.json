{
'project:com/sun/corba/se/impl/activation/ORBD.java':[
191,
198,
205,
212,
219,
],
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
81,
],
'project:com/sun/corba/se/impl/naming/cosnaming/InterOperableNamingImpl.java':[
174,
185,
189,
225,
234,
],
'project:com/sun/corba/se/impl/naming/namingutil/Utility.java':[
56,
58,
],
'project:com/sun/corba/se/impl/orb/DataCollectorBase.java':[
186,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
859,
],
'project:com/sun/corba/se/impl/util/IdentityHashtable.java':[
259,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
1537,
1537,
],
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
853,
855,
918,
920,
],
'project:com/sun/imageio/plugins/bmp/BMPMetadata.java':[
176,
177,
178,
184,
203,
204,
205,
207,
],
'project:com/sun/imageio/plugins/gif/GIFMetadata.java':[
304,
305,
306,
],
'project:com/sun/imageio/plugins/jpeg/JFIFMarkerSegment.java':[
898,
1316,
],
'project:com/sun/imageio/plugins/jpeg/JPEGBuffer.java':[
248,
],
'project:com/sun/imageio/plugins/wbmp/WBMPImageReader.java':[
282,
],
'project:com/sun/imageio/plugins/wbmp/WBMPImageWriter.java':[
228,
],
'project:com/sun/java/swing/plaf/motif/MotifComboBoxUI.java':[
316,
317,
],
'project:com/sun/java/swing/plaf/motif/MotifScrollBarButton.java':[
137,
138,
159,
160,
181,
182,
205,
206,
],
'project:com/sun/java/swing/plaf/windows/WindowsLookAndFeel.java':[
1606,
1607,
1619,
1620,
1634,
1635,
1643,
1644,
1655,
1656,
1665,
1666,
1676,
1677,
1689,
1690,
1707,
1708,
],
'project:com/sun/java/swing/plaf/windows/WindowsScrollBarUI.java':[
438,
443,
],
'project:com/sun/java/swing/plaf/windows/WindowsTreeUI.java':[
93,
],
'project:com/sun/java_cup/internal/runtime/lr_parser.java':[
514,
],
'project:com/sun/jmx/snmp/agent/SnmpGenericObjectServer.java':[
213,
364,
],
'project:com/sun/org/apache/bcel/internal/classfile/ConstantPool.java':[
117,
],
'project:com/sun/org/apache/bcel/internal/util/BCELifier.java':[
245,
],
'project:com/sun/org/apache/bcel/internal/util/Class2HTML.java':[
149,
157,
],
'project:com/sun/org/apache/bcel/internal/util/InstructionFinder.java':[
165,
171,
],
'project:com/sun/org/apache/regexp/internal/RE.java':[
1271,
1272,
1293,
1309,
1336,
],
'project:com/sun/org/apache/regexp/internal/RECompiler.java':[
1455,
],
'project:com/sun/org/apache/regexp/internal/REDebugCompiler.java':[
174,
187,
188,
214,
],
'project:com/sun/org/apache/regexp/internal/REProgram.java':[
144,
148,
],
'project:com/sun/org/apache/xalan/internal/xslt/EnvironmentCheck.java':[
113,
],
'project:com/sun/org/apache/xalan/internal/xslt/Process.java':[
321,
334,
344,
354,
364,
375,
387,
391,
451,
461,
490,
518,
601,
609,
619,
630,
643,
654,
666,
676,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
455,
455,
582,
582,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Stylesheet.java':[
1119,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Whitespace.java':[
276,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/SlotAllocator.java':[
91,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/CurrentNodeListIterator.java':[
144,
192,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/MultiValuedNodeHeapIterator.java':[
245,
245,
246,
249,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/NodeCounter.java':[
203,
204,
218,
219,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/StepIterator.java':[
115,
119,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/BasisLibrary.java':[
351,
351,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/StringValueHandler.java':[
115,
117,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2DOM.java':[
174,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2StAXEventWriter.java':[
340,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMNormalizer.java':[
344,
1437,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredDocumentImpl.java':[
1593,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
1065,
1070,
1072,
1141,
1143,
1187,
1192,
1193,
1198,
1200,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RangeToken.java':[
214,
214,
215,
215,
217,
217,
218,
218,
221,
221,
222,
222,
224,
224,
225,
225,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/SubstitutionGroupHandler.java':[
288,
288,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
3608,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
377,
379,
433,
433,
],
'project:com/sun/org/apache/xerces/internal/util/SymbolHash.java':[
134,
148,
149,
],
'project:com/sun/org/apache/xerces/internal/util/URI.java':[
2177,
],
'project:com/sun/org/apache/xerces/internal/util/XML11Char.java':[
400,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
2966,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeTextReader.java':[
425,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
604,
618,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ExpandedNameTable.java':[
262,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/IncrementalSAXSource_Xerces.java':[
426,
],
'project:com/sun/org/apache/xml/internal/security/utils/XMLUtils.java':[
113,
],
'project:com/sun/org/apache/xml/internal/serialize/BaseMarkupSerializer.java':[
527,
532,
1569,
1575,
1797,
],
'project:com/sun/org/apache/xml/internal/serialize/XML11Serializer.java':[
214,
219,
270,
340,
346,
],
'project:com/sun/org/apache/xml/internal/serialize/XMLSerializer.java':[
1235,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLStream.java':[
1206,
1396,
1404,
2083,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
1110,
1138,
1158,
1980,
1992,
],
'project:com/sun/org/apache/xml/internal/serializer/ToTextStream.java':[
315,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/SystemIDResolver.java':[
206,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/URI.java':[
1638,
],
'project:com/sun/org/apache/xml/internal/utils/QName.java':[
282,
],
'project:com/sun/org/apache/xml/internal/utils/SystemIDResolver.java':[
199,
],
'project:com/sun/org/apache/xml/internal/utils/Trie.java':[
82,
],
'project:com/sun/org/apache/xml/internal/utils/URI.java':[
1658,
],
'project:com/sun/org/apache/xml/internal/utils/XML11Char.java':[
394,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkerFactory.java':[
925,
],
'project:com/sun/org/apache/xpath/internal/compiler/Lexer.java':[
159,
159,
193,
193,
],
'project:com/sun/security/auth/PolicyParser.java':[
905,
907,
908,
],
'project:java/awt/Container.java':[
1434,
1452,
],
'project:java/awt/Font.java':[
2001,
2038,
2078,
],
'project:java/awt/datatransfer/SystemFlavorMap.java':[
464,
466,
471,
],
'project:java/awt/font/TextJustifier.java':[
105,
],
'project:java/awt/font/TextMeasurer.java':[
366,
410,
],
'project:java/io/ObjectOutputStream.java':[
1972,
1982,
2131,
2214,
],
'project:java/io/RandomAccessFile.java':[
1092,
1093,
],
'project:java/lang/AbstractStringBuilder.java':[
1379,
],
'project:java/lang/Character.java':[
5022,
5025,
5061,
5064,
],
'project:java/lang/ConditionalSpecialCasing.java':[
218,
220,
],
'project:java/lang/String.java':[
298,
1789,
],
'project:java/math/BigInteger.java':[
1311,
1312,
1482,
1496,
2709,
2709,
2710,
2712,
2713,
3176,
3176,
3177,
3179,
3180,
],
'project:java/math/MutableBigInteger.java':[
249,
251,
556,
572,
],
'project:java/net/URI.java':[
1711,
1716,
1719,
1722,
2332,
2700,
2776,
2785,
2785,
2786,
],
'project:java/net/URLEncoder.java':[
230,
261,
265,
],
'project:java/text/ChoiceFormat.java':[
194,
],
'project:java/text/DecimalFormat.java':[
2013,
2015,
2020,
2061,
2065,
2071,
2404,
2422,
2430,
2443,
2459,
2552,
2557,
2569,
2574,
],
'project:java/text/DigitList.java':[
311,
],
'project:java/text/MessageFormat.java':[
443,
],
'project:java/text/RBTableBuilder.java':[
533,
577,
],
'project:java/text/SimpleDateFormat.java':[
767,
835,
838,
961,
963,
964,
974,
1388,
1390,
1391,
1406,
],
'project:java/util/Arrays.java':[
598,
600,
782,
784,
],
'project:java/util/Calendar.java':[
1321,
],
'project:java/util/Collections.java':[
825,
],
'project:java/util/DualPivotQuicksort.java':[
124,
126,
131,
132,
172,
176,
178,
181,
187,
216,
246,
249,
251,
564,
566,
571,
572,
612,
616,
618,
621,
627,
656,
686,
689,
691,
996,
1001,
1036,
1038,
1043,
1044,
1084,
1088,
1090,
1093,
1099,
1128,
1158,
1161,
1163,
1468,
1473,
1508,
1510,
1515,
1516,
1556,
1560,
1562,
1565,
1571,
1600,
1630,
1633,
1635,
1943,
1948,
1956,
2056,
2085,
2087,
2092,
2093,
2133,
2137,
2139,
2142,
2148,
2177,
2207,
2210,
2212,
2583,
2612,
2614,
2619,
2620,
2660,
2664,
2666,
2669,
2675,
2704,
2734,
2737,
2739,
],
'project:java/util/Formatter.java':[
2529,
],
'project:java/util/Hashtable.java':[
403,
],
'project:java/util/LinkedList.java':[
456,
],
'project:java/util/Random.java':[
217,
],
'project:java/util/concurrent/ConcurrentHashMap.java':[
385,
],
'project:java/util/concurrent/ConcurrentLinkedDeque.java':[
361,
362,
362,
365,
365,
394,
395,
395,
398,
398,
476,
497,
538,
558,
559,
572,
592,
593,
751,
752,
752,
755,
755,
776,
777,
777,
780,
780,
1178,
1179,
1179,
1182,
1182,
1194,
],
'project:java/util/concurrent/ConcurrentLinkedQueue.java':[
348,
348,
351,
351,
365,
368,
375,
385,
392,
410,
417,
547,
560,
560,
563,
563,
],
'project:java/util/concurrent/ForkJoinPool.java':[
764,
764,
768,
768,
768,
768,
771,
823,
838,
844,
852,
865,
1142,
1142,
1165,
1181,
],
'project:java/util/concurrent/ForkJoinTask.java':[
238,
],
'project:java/util/concurrent/ForkJoinWorkerThread.java':[
551,
712,
719,
722,
727,
730,
789,
822,
],
'project:java/util/concurrent/LinkedBlockingDeque.java':[
994,
],
'project:java/util/concurrent/LinkedTransferQueue.java':[
625,
626,
634,
634,
662,
669,
669,
672,
676,
812,
815,
857,
861,
876,
877,
880,
974,
975,
979,
1001,
1002,
1003,
1004,
],
'project:java/util/prefs/FileSystemPreferences.java':[
863,
864,
879,
880,
],
'project:java/util/regex/Pattern.java':[
1367,
1378,
1383,
1513,
3248,
3250,
],
'project:javax/management/ObjectName.java':[
1885,
],
'project:javax/management/modelmbean/DescriptorSupport.java':[
1186,
],
'project:javax/naming/NameImpl.java':[
284,
294,
385,
388,
409,
412,
426,
433,
438,
441,
],
'project:javax/naming/ldap/Rdn.java':[
646,
659,
],
'project:javax/swing/JComponent.java':[
2267,
5457,
],
'project:javax/swing/JMenu.java':[
1508,
],
'project:javax/swing/JTree.java':[
3163,
],
'project:javax/swing/LookAndFeel.java':[
435,
],
'project:javax/swing/plaf/basic/BasicFileChooserUI.java':[
1069,
1073,
1086,
1090,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
1944,
],
'project:javax/swing/text/AbstractDocument.java':[
1135,
1226,
],
'project:javax/swing/text/DefaultHighlighter.java':[
68,
],
'project:javax/swing/text/GlyphView.java':[
1029,
1041,
],
'project:javax/swing/text/MaskFormatter.java':[
513,
749,
775,
],
'project:javax/swing/text/StringContent.java':[
233,
248,
283,
],
'project:javax/swing/text/html/AccessibleHTML.java':[
2511,
2520,
],
'project:javax/swing/text/html/FrameSetView.java':[
195,
],
'project:javax/swing/text/html/HTMLDocument.java':[
2931,
2932,
2937,
2940,
],
'project:javax/swing/text/html/Map.java':[
365,
],
'project:javax/swing/tree/DefaultTreeSelectionModel.java':[
718,
],
'project:javax/swing/tree/FixedHeightLayoutCache.java':[
1212,
1255,
1302,
1409,
],
'project:javax/xml/soap/MimeHeaders.java':[
113,
164,
],
}
